# SBG Ellipse Messages
Header header

# Time since sensor is powered up (us)
uint32 time_stamp

# Magnetometer output (X, Y, Z)
geometry_msgs/Vector3 mag

# Accelerometer output (X, Y, Z) in m/s2
geometry_msgs/Vector3 accel

# Status
SbgMagStatus status
